metabench.cmake

```diff
@@ -426,10 +434,10 @@ file(WRITE ${METABENCH_RB_PATH}
 "      IO.write(cpp_file, code)                                                                           \n"
 "      results = repetitions.times.map { build(target) }                                                  \n"
 "      datum = {}                                                                                         \n"
-"      datum['compilation_times'] = results.map { |r| r['compilation_time'] }                             \n"
-"      datum['link_times']        = results.map { |r| r['link_time'] }                                    \n"
-"      datum['executable_size']   = results.map { |r| r['executable_size'] }.first                        \n"
-"      datum['execution_times']   = results.map { |r| r['execution_time'] }                               \n"
+"      datum['compilation_times'] = results.map { |r| (scale)*r['compilation_time'] }                     \n"
```

Comment:
Instead, would it make sense to pass `datum['scale']` to the JavaScript and do the scaling over there?